plugins/chat/spec/models/chat_mention_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
```

Comment:
I think it should be in `chat_message_spec`. Here you are in `chat_mention_spec` describing ChatMessage model, it's a little bit unexpected

Reply:
Oh, that's a mistake, thank you for catching, it should be in ChatMessage spec for sure.